src/aggregators/ssajump.jl

```diff
@@ -36,11 +34,23 @@ end
 # execute_jumps!
 # generate_jumps!
 
+@inline function makewrapper(::Type{T}, aff) where T
```

Comment:
Did you paste this into the file instead of merging master into your branch? (Or rebasing?) 

Don't do that, it messes up the git history among other issues. Can you remove any such cut and pastes and instead merge master in the branch?

Reply:
I don't recall pasting this. I've always merged with master.

Even on my side I get this `git-diff`:

```
(coevolve-ii) JumpProcesses % git diff origin/master -- src/aggregators/ssajump.jl
diff --git a/src/aggregators/ssajump.jl b/src/aggregators/ssajump.jl
index a5904e8..681cd99 100644
--- a/src/aggregators/ssajump.jl
+++ b/src/aggregators/ssajump.jl
@@ -17,9 +17,7 @@ An aggregator interface for SSA-like algorithms.
 
 ### Optional fields:
 
-  - `dep_gr`             # dependency graph, dep_gr[i] = indices of reactions that should
-
-    # be updated when rx i occurs.
+  - `dep_gr`             # dependency graph, dep_gr[i] = indices of reactions that should be updated when rx i occurs.
 """
 abstract type AbstractSSAJumpAggregator{T, S, F1, F2, RNG} <: AbstractJumpAggregator end
```

And then with `git-blame` it shows that you added the line:

```sh
(coevolve-ii) JumpProcesses % git blame src/aggregators/ssajump.jl | grep '@inline function make'
73b947ce (Sam Isaacson     2023-07-10 10:13:36 -0400  37) @inline function makewrapper(::Type{T}, aff) where T
```

Reply:
Weird usually I feel like GitHub doesn’t render such things in the online diff. In any case sounds good!